README.md

```diff
@@ -1,17 +1,42 @@
 # govuk-dependencies
 
-## Sinatra Application For Dependency Reporting
+A tool for viewing all of the pull requests made by Dependabot to Gov.Uk repos, as well as for
+sending slack messages to GDS Teams to remind them of what PRs Dependabot has open against their
+applications in order to prevent a large backlog of PRs.
 
+## Screenshots
 
-### Start the application
+![screenshot](https://user-images.githubusercontent.com/976254/35578664-b0617926-05dc-11e8-9281-3b307a8792c4.png)
+
+## Live examples
+
+- [https://govuk-dependencies.herokuapp.com/](https://govuk-dependencies.herokuapp.com/)
+
+## Technical documentation
+
+This is a Sinatra application that uses the GitHub API in order to get a list of of PRs made
+by Dependabot and groups them in various ways:
+
+- By application
+- By team
+- By gem
+
+### Dependencies
+
+- [octokit/octokit.rb](https://github.com/octokit/octokit.rb) - Used for interacting with the GitHub API
+
+### Running the application
 
 `bundle exec rackup`
 
-### Running tests
+Running this will start your application at [localhost:9292](localhost:9292)
 
-`bundle exec rake`
+### Running the test suite
 
-### Running ruby linting
+`bundle exec rake`
 
-`bundle exec govuk-lint-ruby`
+### Rate limiting
 
+If you find yourself being rate limited by GitHub - you can define the `GITHUB_TOKEN` environment variable.
+This needs to be a token generated from GitHub, however as the repositories are all public it needs no special
+permissions.
```

Comment:
Would also mention the constants that need to be set
SLACK_WEBHOOK_URL
GITHUB_TOKEN

Reply:
Ah yeah - forgot `SLACK_WEBHOOK_URL`